src/test/mina_automation/executor.ml

```diff
@@ -75,49 +73,111 @@ module Make (P : AppPaths) = struct
     [%log debug] "Executing mina application"
       ~metadata:[ ("app", `String path) ]
 
-  let run_from_debian ?(prefix = "") ~(args : string list) ?env () =
+  let run_from_local_in_background ~(args : string list) ?env () =
```

Comment:
What's preventing us from having a generic `in_background` that we could plug in any configuration from {local, debian, dune, docker}? These helpers look mostly identical